src/content/tools/widget-previewer.md

```diff
@@ -17,16 +17,39 @@ in it, and customize a preview, see the following sections.
 
 :::version-note
 The Flutter Widget Preview requires Flutter version 3.35 or
-higher.
+higher. IDE support requires Flutter version 3.38 or higher.
 
 Please be aware that this is an **experimental feature**
 available in the Flutter stable channel. The APIs are not
-stable and *will change*. This guide is for the current
+stable and _will change_. This guide is for the current
 early access version, and you should expect future updates
 to introduce breaking changes.
 :::
 
-## Start the previewer
+## Opening the previewer
+
+### IDEs
+
+As of Flutter 3.38, Android Studio, Intellij, and Visual
+Studio Code automatically start the Flutter Widget Previewer
+on launch.
+
+#### Android Studio
+
+// TODO
```

Comment:
I need to get an updated version of the Android Studio / Intellij plugin to get these screenshots.